lib/pysquared/functions.py

```diff
@@ -32,7 +32,7 @@ def debug_print(self, statement: Any) -> None:
         if self.debug:
             print(co("[Functions]" + str(statement), "green", "bold"))
 
-    def __init__(self, cubesat: Satellite) -> None:
+    def __init__(self, cubesat: Satellite, Config: Config) -> None:
```

Comment:
```suggestion
    def __init__(self, cubesat: Satellite, config: Config) -> None:
```

Looking for consistent capitalization for variable names.